README.md

```diff
@@ -23,6 +23,16 @@ https://api.datapunt.amsterdam.nl/afvalophaalgebieden/search/?x=121394.0&y=48738
 # Developing
 Use `docker-compose` to start a local database.
 
+The https proxy config is prevent local development right now.
```

Comment:
Is it a better idea to set this (environmental) variable when building for a specific environment, in our Ansible scripts, instead of here?

Reply:
Hmm, I'm going to refrain from doing this in this PR. I'll try to get a fix right after this one.

Reply:
That's okay.